internal/pipe/brew/brew.go

```diff
@@ -150,6 +155,14 @@ func doPublish(ctx *context.Context, formula *artifact.Artifact, cl client.Clien
 	return cl.CreateFile(ctx, author, repo, content, gpath, msg)
 }
 
+func getClient(ctx *context.Context, brew config.Homebrew, cl client.Client, token string) (client.Client, error) {
+	cl, err := client.NewIfToken(ctx, cl, brew.Tap.Token)
+	if len(brew.Tap.GitURL) > 0 && ctx.TokenType != context.TokenTypeGitLab {
```

Comment:
why != gitlab? 🤔

Reply:
A temporary bandaid I forgot to commit the real fix for.  Because the raw git client does not use tokens the current `NewIfToken` function would either give it a github client which would pass the test but fail in real world or if I didn't put the second expression in the if statement give it the gitclient during testing which would cause the test to fail. I have a better solution but forgot to commit it.